sql/core/src/main/scala/org/apache/spark/sql/expressions/Window.scala

```diff
@@ -159,6 +159,17 @@ object Window {
   }
 
   /**
+   * @param start boundary start, inclusive. The frame is unbounded if this evals to
+   *              the minimum long value (`Window.unboundedPreceding`).
```

Comment:
`.. the minimum long value ...` this is not true now, we use a special value `functions.unboundedPreceding`